cdisc_rules_engine/data_service/sql_data_preprocessor.py

```diff
@@ -673,14 +468,15 @@ def _build_relrec_catalog(self) -> Dict[str, Any]:
             LIMIT 1
         """
 
-        self.pgi.execute_sql(check_query)
-        result = self.pgi.fetch_one()
+        self.data_service.pgi.execute_sql(check_query)
```

Comment:
What are relrec, supp... still doing here? What do they do? We do those joins as they're required